app/routes/dashboards/dashboard.js

```diff
@@ -534,30 +534,30 @@ export default Ember.Route.extend({
                     },
                 ]
             },
-            resultsList: {
+            search: {
```

Comment:
There are still a few places that are transitioning to the `resultsList` dashboard: 
- https://github.com/birdbrained/share-analytics/blob/master/app/components/contributors-widget/component.js#L88
- https://github.com/birdbrained/share-analytics/blob/master/app/components/object-detail-widget/template.hbs#L11

Can you double check as well to make sure there are no transitions to `resultsList` instead of the `search` dashboard?